cmd/empire/factories.go

```diff
@@ -93,9 +93,17 @@ func newScheduler(db *empire.DB, c *cli.Context) (scheduler.Scheduler, error) {
 		return nil, err
 	}
 
-	s, err := newMigrationScheduler(db, c)
-	if err != nil {
-		return nil, err
+	var s scheduler.Scheduler
+	if c.String(FlagDefaultScheduler) == "cloudformation" {
```

Comment:
Tweaked this a bit so that the default (opt out) bypasses cloudformation.MigrationScheduler as well.